sphinx/util/typing.py

```diff
@@ -316,7 +382,10 @@ def stringify_annotation(
         else:
             return annotation
     elif isinstance(annotation, TypeVar):
-        if annotation_module_is_typing and mode in {'fully-qualified-except-typing', 'smart'}:
+        if (
+                annotation_module_is_typing and
+                mode & (RenderMode.smart | RenderMode.fully_qualified_except_typing)
+        ):
```

Comment:
Indentation fixup:

```suggestion
        if (
            annotation_module_is_typing and
            mode & (RenderMode.smart | RenderMode.fully_qualified_except_typing)
        ):
```

Reply:
I think it's PyCharm that introduced this. I'll change this tomorrow